client/src/main.jsx

```diff
@@ -1,35 +1,32 @@
-import ReactDOM from 'react-dom/client';
-import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import ReactDOM from 'react-dom/client'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 import './output.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
 
-import App from './App.jsx';
-import Home from './pages/Home';
-import ErrorPage from './pages/ErrorPage';
-import ResultPage from './pages/ResultPage';
-import Donation from './pages/Donation';
```

Comment:
donation page should not be deleted. Or do you integrated donation page somewhere?